### PR TITLE
Adding multiple Devices and Width UITests 

### PR DESCRIPTION
**Pull request by reyesrico**
Adding different Devices and Width UITests
Improving timeouts for Client and Server command_map tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,8 +43,9 @@
     "@types/chai-subset": "^1.0.29",
```
Comment (eanders-MS) [inline package.json]: I think all @types can go into the devDependencies section.

File: package.json [hunk 2]
```diff
@@ -25,9 +25,7 @@
   "author": "Microsoft Corp",
   "license": "MIT",
   "dependencies": {
-    "@types/markdown-it": "0.0.2",
-    "@types/react-redux": "^4.4.38",
-    "@types/redux": "^3.6.0",
+    "@types/react": "^15.0.21",
```
Comment (eanders-MS) [inline package.json]: Also move @types/react.

File: package.json (hunk 1 above)
Comment (eanders-MS) [inline package.json]: @danmarshall schooled me on this. Some do need to go in dependencies!

File: test/uitest.js [hunk 3]
```diff
@@ -10,61 +10,53 @@ let nightmare = Nightmare({
...
+
+					result = yield nightmare.goto(testUrl)
+						.viewport(width, 768)
+						.wait(2000)
+						.type('.wc-textbox input', keys[cmd_index])
+						.click('.wc-send')
+						.wait(3000)
+						.evaluate(commands[keys[cmd_index]].client)
+
+					result ? console.log("\x1b[32m", tab + tab + result) : console.log("\x1b[31m", tab + tab + result);
```
Comment (danmarshall) [inline test/uitest.js]: can you add a function to wrap the `"\x1b[32m"` stuff

File: package.json [hunk 4]
```diff
@@ -64,8 +63,8 @@
     "npm-run-all": "^3.1.2",
     "remote": "^0.2.6",
     "source-map-loader": "^0.2.1",
-    "typescript": "^2.2.2",
+    "typescript": "2.2.2",
```
Comment (GeekTrainer) [inline package.json]: Why the pin to 2.2.2?

Comment (danmarshall) [inline package.json]: On a fresh clone, it would pull the latest, 2.3.1 and we saw some build errors.

**Pull request merged**